include/libcyphal/transport/udp/media.hpp

```diff
@@ -25,7 +27,12 @@ class IMedia
     IMedia& operator=(const IMedia&)     = delete;
     IMedia& operator=(IMedia&&) noexcept = delete;
 
-    // TODO: Add methods here
+    /// @brief Get the maximum transmission unit (MTU) of the UDP media.
+    ///
+    /// This value may change arbitrarily at runtime. The transport implementation will query it before every
+    /// transmission on the port. This value has no effect on the reception pipeline as it can accept arbitrary MTU.
+    ///
+    CETL_NODISCARD virtual std::size_t getMtu() const noexcept = 0;
```

Comment:
This is a public API and nothing catastrophic happens if the value isn't used. Please omit `CETL_NODISCARD` value

Reply:
ok, I will do it. But there is separate issue about revising everything in terms of "to discard or not to discard" question.